test/others/test_utils.py

```diff
@@ -17,7 +20,15 @@
 from haystack.utils.reflection import retry_with_exponential_backoff
 from haystack.utils.context_matching import calculate_context_similarity, match_context, match_contexts
 
-from ..conftest import DC_API_ENDPOINT, DC_API_KEY, MOCK_DC, SAMPLES_PATH, deepset_cloud_fixture
+from ..conftest import (
+    DC_API_ENDPOINT,
+    DC_API_KEY,
+    MOCK_DC,
+    SAMPLES_PATH,
+    deepset_cloud_fixture,
+    fail_at_version,
+    haystack_version,
+)
 
 TEST_CONTEXT = context = """Der Merkantilismus förderte Handel und Verkehr mit teils marktkonformen, teils dirigistischen Maßnahmen.
```

Comment:
By the way, do you know if this is a bug or something exotic? Looks like invalid syntax to me :thinking:

Reply:
`=` can be used consecutively so the syntax is correct but my guess somebody wanted to rename `context` to `TEST_CONTEXT` and forgot to remove `context`. We can safely remove `context = `